plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -664,16 +664,17 @@ public void testVarbinary()
     @Test
     public void testDateMapping()
     {
-        DateTests(zone -> trinoCreateAsSelect("nl_date_" + zone));
+        dateTests(zone -> trinoCreateAsSelect("date_" + zone));
+        dateTests(zone -> trinoCreateAndInsert("date_" + zone));
```

Comment:
preexisting, but zone doesn't need to be part of test table name
(and some zone names are not suitable for table names either)

see https://github.com/trinodb/trino/blob/714e70e0bb80f844536aca1222f603a9f4d09b07/plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java#L981-L984